.buildkite/docker-compose.yml

```diff
@@ -47,8 +48,15 @@ services:
       args:
         PYTHON_VERSION: "3.7"
 
+  # a service that does nothing to use as a default depends_on for the runners
+  empty-service:
+    image: alpine:3.10
```

Comment:
This image could be the `python:${PYTHON_VERSION}`, i.e., the base image of `stellargraph-ci-runner` so it doesn't have to pull down an extra image? Or perhaps a [busybox](https://hub.docker.com/_/busybox?tab=tags) tag which are typically < 1MB in size (unfortunately `scratch` is a reserved name and won't work in `docker-compose`)

Reply:
I chose this because it's the base image for many of our other tools, and so is likely to exist on the machines already. It's only a few megabytes too. There's no `PYTHON_VERSION` variable currently available, but it could potentially be.